shopinvader/models/product_product.py

```diff
@@ -59,12 +63,26 @@ def _add_to_cart_allowed(self, backend, partner=None):
             self.shopinvader_bind_ids.filtered(lambda x: x.backend_id == backend)
         )
 
-    def _get_invader_variant(self, backend, lang):
+    def _get_invader_variant(self, backend, lang, safe_default=False):
         """Retrieve invader variant by backend and language.
 
         :param backend: backend recordset
         :param lang: lang code
+        :param safe_default: true to fallback to backend lang
         """
+        variant = self._find_invader_variant(backend, lang)
+        if not variant and safe_default:
+            lang = self.shopinvader_bind_ids.filtered(
+                lambda x: x.backend_id == backend
+            ).lang_id.code
```

Comment:
this is broken w/ more than one lang!
```suggestion
            # TODO: we should probably have a new field `default_lang_id` on the backend
            lang = fields.first(self.shopinvader_bind_ids.filtered(
                lambda x: x.backend_id == backend
            )).lang_id.code
```

Reply:
It's a Many2one isn't it? Only one lang can be stored here
![image](https://user-images.githubusercontent.com/77412816/190096748-08afbe12-5bb5-45c4-9f51-7aead551478a.png)

Reply:
See comment above